src/components/screens/wallet/balanceChart.js

```diff
@@ -1,63 +1,60 @@
 import React from 'react';
 import { withTranslation } from 'react-i18next';
-import { Line as LineChart } from 'react-chartjs-2';
 import Box from '../../toolbox/box';
 import BoxHeader from '../../toolbox/box/header';
 import BoxEmptyState from '../../toolbox/box/emptyState';
-import styles from './balanceChart.css';
 import * as ChartUtils from '../../../utils/balanceChart';
 import { tokenMap } from '../../../constants/tokens';
 import i18n from '../../../i18n';
+import Chart from '../../toolbox/Chart';
```

Comment:
💅 Our naming convention is to start with lower-case. To be updated in all places
```suggestion
import Chart from '../../toolbox/chart';
```